mage_ai/orchestration/db/models/schedules.py

```diff
@@ -287,20 +289,32 @@ def create_or_update_batch(self, trigger_configs: List[Trigger]):
                 traceback.print_exc()
                 continue
 
+            last_enabled_at = trigger_config.last_enabled_at
+            if trigger_config.status == ScheduleStatus.ACTIVE and \
+                    trigger_config.last_enabled_at is None:
+                last_enabled_at = datetime.now(tz=pytz.UTC)
+                trigger_config.last_enabled_at = last_enabled_at
+                add_or_update_trigger_for_pipeline_and_persist(
+                    trigger_config,
+                    trigger_config.pipeline_uuid,
+                )
+
             kwargs = dict(
+                last_enabled_at=last_enabled_at,
                 name=trigger_config.name,
                 pipeline_uuid=trigger_config.pipeline_uuid,
+                schedule_interval=trigger_config.schedule_interval,
                 schedule_type=trigger_config.schedule_type,
+                settings=trigger_config.settings,
+                sla=trigger_config.sla,
                 start_time=trigger_config.start_time,
-                schedule_interval=trigger_config.schedule_interval,
                 status=trigger_config.status,
                 variables=trigger_config.variables,
-                sla=trigger_config.sla,
-                settings=trigger_config.settings,
             )
 
             if existing_trigger:
                 if any([
+                    existing_trigger.last_enabled_at != kwargs.get('last_enabled_at'),
```

Comment:
> i think you may need to update the code here too: https://github.com/mage-ai/mage-ai/blob/master/mage_ai/orchestration/db/models/schedules.py#L238
> 
> ```
> def create_or_update_batch(self, trigger_configs: List[Trigger]):
> ```

@dy46 I think the link to the code line may have changed since it was added, but i think you were referring to the line above. Which part were you referring to specifically for updating the code? Not the parts updated here?